docs/_documentations/troubleshooting.md

```diff
@@ -231,3 +232,71 @@ Issue link:
 If you turn off `auto build` for a Node.js project when you run Codewind locally, it has no effect. Changes you make to your code automatically start or restart a build even though `auto build` is disabled.
 
 ***
+# Appsody with Codewind
+
+<!--
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind-docs/issues/64
+18.10:
+-->
+## Node.js and Swift templates remain in the starting state
+Node.js and Swift templates remain in the starting state because they do not connect with the server.
+
+<!--
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind-docs/issues/64
+18.10:
+-->
+## A project build error appears before you create an initial project
+If you use Eclipse and either Java MicroProfile or Spring Appsody templates, you might receive a `Project build error: Non-resolvable parent POM` error before you create the initial project.
+
+**Workaround** Complete the following instructions to work around the error:
+1. From the New Codewind Project Wizard, select a Java-based Appsody template.
+2. Complete the prompts that the wizard presents.
+3. After the project appears in the Codewind Explorer view, right-click the project and select **Show Log Files**>**Show All**.
+4. If your `.m2` cache is empty, or if you have not previously created a Java Appsody project, the dependencies will download, and the `[Container] Installing parent dev.appsody` message appears.
+5. Wait until the cache completes. You can wait until the `Liberty defaultServer` starts or when the Project status is `Running`.
```

Comment:
I think `Liberty defaultServer` is only applicable to the Microprofile template, Spring is not using Liberty